rest/v9/guild.ts

```diff
@@ -29,7 +31,7 @@ export interface APIGuildCreateOverwrite extends RESTPutAPIChannelPermissionJSON
 }
 
 export type APIGuildCreatePartialChannel = Partial<
-	Pick<APIChannel, 'type' | 'topic' | 'nsfw' | 'bitrate' | 'user_limit' | 'rate_limit_per_user'>
+	Pick<APIVoiceChannel & APITextChannel, 'type' | 'topic' | 'nsfw' | 'bitrate' | 'user_limit' | 'rate_limit_per_user'>
```

Comment:
Why is this using `APIVoiceChannel & APITextChannel`? where's my stores and categories? This should be an `Exclude<APIChannel, APIDMChannel | APIGroupDMChannel>` or similar